README.adoc

```diff
@@ -245,40 +245,39 @@ Create the `FaultToleranceTest` class in the
 include::finish/src/test/java/it/io/openliberty/guides/faulttolerance/FaultToleranceTest.java[tags=ft_testing;!javadoc]
 ----
 
-The `@BeforeClass` annotation is placed on a method that executes before any of the test cases. In
+The `@BeforeClass` annotation is placed on a method that executes before any of the other test cases. In
 this case, the `oneTimeSetup()` method retrieves the port number for the Open Liberty server and 
 builds a base URL string that is used throughout the tests.
 
-The `@Before` and `@After` annotations are placed on methods that execute before and after every 
-test case. These methods are generally used to perform any setup and teardown tasks. In this case,
-the setup method creates a JAX-RS client which makes HTTP requests to the `inventory` service. This
-client must also be registered with a JSON-P provider (JsrJsonpProvider) to process JSON resources. 
-The teardown method simply destroys this client instance as well as the HTTP responses, resets the
+The `@Before` and `@After` annotations are placed on methods that execute before and after any 
```

Comment:
=> same here: "The `@Before` and `@After` annotations places on methods that execute before and after any other test case."

Reply:
"Place the `@Before` and `@After` annotations on methods that execute before and after any other test case."?

Reply:
=> The `@Before` and `@After` annotations indicates that this method executes before or  after the other test case. (what do you think?)

Reply:
Yes, we can go with "The `@Before` and `@After` annotations indicate that this method executes before or after the other test case." <--I just changed the verb to "indicate" to make it plural (since we're talking about 2 annotations).